tests/integrationtest/t/executor/infoschema_reader.test

```diff
@@ -183,9 +183,6 @@ select * from information_schema.`METRICS_TABLES` where table_name='tidb_qps';
 # TestTableConstraintsTable
 select * from information_schema.TABLE_CONSTRAINTS where TABLE_NAME='gc_delete_range';
 
-# TestTableSessionVar
-select * from information_schema.SESSION_VARIABLES where VARIABLE_NAME='tidb_retry_limit';
```

Comment:
Could we have negative tests (i.e. that the tables does not exists) for the four old information_schema tables?